src/jit/SimdArmInl.h

```diff
@@ -0,0 +1,287 @@
+/*
+ * Copyright (c) 2023-present Samsung Electronics Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Only included by jit-backend.cc */
+
+constexpr uint8_t getUpperOfQRegister(uint8_t reg)
+{
+    return reg + 1;
+}
+
+namespace SimdOp {
+
+#if (defined SLJIT_CONFIG_ARM_THUMB2 && SLJIT_CONFIG_ARM_THUMB2)
+constexpr uint8_t sizeOffset = 20;
+constexpr uint8_t singleSizeOffset = 18;
+constexpr uint32_t unsignedBit = 0b1 << 28;
+
+enum Type : uint32_t {
+    vaddInt = 0xef000840,
+    vmlal = 0xef800800,
+    vmulInt = 0xef000950,
+    vmull = 0xef800c00,
+    vnegT1 = 0xffb103c0,
```

Comment:
T1?

Reply:
This operation has T1 and T2 form. Here the T1 is used.

Reply:
There is only one Advanced SIMD variant.